classes/basement/rootrecipe.yaml

```diff
@@ -41,6 +41,9 @@ depends:
           - name: devel::help2man
             use: [tools]
             forward: True
+          - name: devel::automake
```

Comment:
Hmm, `automake` is part of the `autotools` tool already. Why do we need it again separately? I would only do this if there is a good reason...

BTW, the "remove automake from host prequisites" commit which adds this as tool dependency, does not really does what the commit message says.